### ISSUE TITLE
Question: when should we release 005?

### ISSUE THREAD

**Issue by johnbelamaric** [coredns/coredns#495]
Lots of changes since 004, and I am working on an updated K8s article. Would be nice to have 005 out before publishing it.

@miekg what do you want to see done before release of 005?

@chrisohaver is working on a config option for the k8s middleware to make configuration the reverse zones simpler. Should be ready in a couple days if he can get the time. That would be nice to have in 005 too.

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "[miekg/coredns] Question: when shou..." ]
>Lots of changes since 004, and I am working on an updated K8s article. Would be nice to have 005 out before publishing it.
>
>@miekg what do you want to see done before release of 005?

I would like to merge my proxy stuff, however that is slow going due to work
(and LEGO :-) ). I'll try complete it this week, otherwise we release 005
without it.

>@chrisohaver is working on a config option for the k8s middleware to make configuration the reverse zones simpler. Should be ready in a couple days if he can get the time. That would be nice to have in 005 too.

Ok, we can wait for that.

**Comment by johnbelamaric** [comment]
I am thinking we should have a coredns Docker hub repo and put an automated build in there.

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [miekg/coredns] Question: when ..." ]
>I am thinking we should have a coredns Docker hub repo and put an automated build in there.

Yes, last time I looked for SkyDNS they automated docker build stuff was still
pretty wonky. But yes we should set something up there.

**Comment by miekg** [comment]
Think we're unblocked for a 005 release?

Can do the docker thing for 006

**Comment by johnbelamaric** [comment]
Think so. I'd Like to verify a couple things tomorrow just to be sure.

**Comment by miekg** [comment]
Ack. Shouldn't we move those "couple [of] things" in to tests?

**Comment by johnbelamaric** [comment]
Yeah. I think they already are but going to double check.

**Comment by johnbelamaric** [comment]
Ok, looks good. @chrisohaver is going to beef up the integration tests but I don't think 005 has to wait for that.

**Comment by johnbelamaric** [comment]
Actually, wait. Still seeing an issue.

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [miekg/coredns] Question: when ..." ]
>Actually, wait. Still seeing an issue.

No release will be perfect.. But I haven't done anything yet.

**Comment by johnbelamaric** [comment]
Yup. #510 though needed fixing. Assuming it goes through test we can merge and release.

**Comment by johnbelamaric** [comment]
I built it and pushed it to docker.io/coredns/coredns repo.
